### PR TITLE
Return multiple results as array rather than just the first one.

### PR DESCRIPTION
**Pull request by Pes8**
As requested in pull request #68  I've change the code to return an array of array instead of a Collection of array

### CODE REVIEW

File: src/Geocoder.php [hunk 1]
```diff
@@ -73,11 +73,36 @@ public function setCountry(string $country)
     public function getCoordinatesForAddress(string $address): array
     {
         if (empty($address)) {
-            return $this->emptyResponse();
+            return $this->emptyResponse()[0];
```
Comment (freekmurze) [inline src/Geocoder.php]: Instead of using a key here, let's just add a named method `emptySingleResponse`

File: src/Geocoder.php [hunk 2]
```diff
@@ -102,37 +127,38 @@ public function getAddressForCoordinates(float $lat, float $lng): array
...
-            'lng' => $response->results[0]->geometry->location->lng,
-            'accuracy' => $response->results[0]->geometry->location_type,
-            'formatted_address' => $response->results[0]->formatted_address,
-            'viewport' => $response->results[0]->geometry->viewport,
-            'address_components' => $response->results[0]->address_components,
-            'place_id' => $response->results[0]->place_id,
-        ];
+        $locations = [];
+
+        foreach ($response->results as $result) {
```
Comment (freekmurze) [inline src/Geocoder.php]: Let's make it more expressive by using `array_map`

**Comment by Pes8** [comment]
Any chances it will be merged? I need this features ASAP

**Comment by Pes8** [comment]
Any news ? I'm using my local repo but it should be great to see the PR merged with the official repository :/

**Comment by freekmurze** [comment]
Does this PR contain breaking changes? It seems to me that the response format has changed.

Could you also update the readme?

**Comment by Pes8** [comment]
@freekmurze no breaking changes, only a new feature as the title. There is a new method `getAllCoordinatesForAddress` that returns an array of results instead of the first result. The rest it's all the same (the PR was made 2 months ago, I don't know if after the basecode is changed but I see that all the tests passed)

I can try to update the README but I'm not a native english speaker...

**Comment by Pes8** [comment]
@freekmurze DONE. Updated also the readme...

**Comment by freekmurze** [comment]
Thanks!

**Pull request merged**